lib/web_ui/dev/supported_browsers.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:test_api/src/backend/runtime.dart';
+
+import 'browser.dart';
+import 'chrome.dart';
+import 'chrome_installer.dart';
+import 'common.dart';
+import 'firefox.dart';
+import 'firefox_installer.dart'; // ignore: implementation_imports
+
+/// Class that has mappings and utility methods for the browsers tests are
+/// supported.
+class SupportedBrowsers {
+  final List<BrowserArgParser> argParsers =
+      List.of([ChromeArgParser(), FirefoxArgParser()]);
+
+  final List<String> supportedBrowserNames = ['chrome', 'firefox'];
+
+  final Map<String, Runtime> supportedBrowsersToRuntimeMap = {
```

Comment:
Hmmmmm. Indeed we can easily have browsers we implemented and not supporting. I think that is an issue observed a lot in big projects.

We can implement many browsers. On the other hand supporting the tests actively for them is a different story since it takes more than just writing a class that implements Browser and an installer. May be some tests will turn out very flaky for example and we will remove the corresponding browser from the "supportedBrowsers" for while.

I am ok for other name suggestions but I will like to make it explicit enough that future developers can easily see the difference.

I can remove Map, on the other hand that name does not suggest Map as a data structure but map as a verb. What it does it mapping runtimes to browsers.

Reply:
Removed map.